### PR TITLE
Added Scroll To Top Button

### PR DESCRIPTION
**Pull request by anupamishra333**

### CODE REVIEW

**Comment by sifa123** [comment]
I can see the button but it's not working. @anupamishra333  please check this again

**Comment by anupamishra333** [comment]
@sifa123 I tried in my pc and it is working with the same code 

https://user-images.githubusercontent.com/66557767/126858160-274cceb1-eb87-4f3e-9fd1-2ece066d9fbb.mp4

However, I can add another code but it doesn't have fading feature.

File: index.html [hunk 1]
```diff
@@ -1,154 +1,192 @@
...
+      </p>
+    </div>
+  </footer>
+
+  <!-- scroll to top button -->
+  <button id="btnScrollToTop">
+    <img src="images/scrtotop.jfif" alt="scrbtn">
+  </button>
+  
+  <script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.2/jquery.min.js"></script>
```
Comment (sifa123) [inline index.html:173]: You missed **https:** in this line, write like this  
```<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.0.2/jquery.min.js"></script>```
That's why it wasn't working.

Comment (anupamishra333) [reply index.html]: oh sorry, my bad. Added

**Comment by anupamishra333** [comment]
kindly label the issue as DCP 21
